sentences/nl/_common.yaml

```diff
@@ -50,18 +50,18 @@ lists:
         out: "fahrenheit"
 expansion_rules:
   lamp: "[de|het] (lamp[en]|licht[en]|verlichting)"
-  ventilator: "[de] (ventilator[s|en])"
+  ventilator: "[de] (ventilator[s|en]|fan[s])"
   afdekking: "[de|het] (gordijn[en]|vitrage[s]|jaloezie[ën]|rolluik[en]|screen[s]|luxaflex|scherm[en])"
   name: "[de|het] {name}"
   area: "[de|het] {area}"
   doe: "(zet[ten]|mag|mogen|doe[n]|verander[en]|maak|maken|schakel[en])"
   zou: "(kan|kun[t]|zal|zou) (je|jij|u)"
   naar: "(naar|op)"
-  helderheid: "[de] (helderheid|felheid|intensiteit)"
+  helderheid: "[de] (helderheid|felheid|intensiteit|lichtsterkte)"
   brightness: "{brightness}[%|procent]"
   temperature: "{temperature}[°|graden] [{temperature_unit}]"
   warm: "(warm|heet|koud|koel)"
-  open: "(open[en]|omhoog|naar boven)"
+  open: "(open[en]|omhoog|naar boven|opwaarts)"
   dicht: "(dicht|omlaag|naar beneden|sluiten)"
```

Comment:
if you want to add `opwaarts`, should we nog also add `neerwaarts` as well in the open expansion rule?